File: frontend/app/src/main/java/com/example/trenucizapamcenje/models/Offer.kt
package com.example.trenucizapamcenje.models

data class Offer(
    val _id: String,
    val offerImageUrl: String,
    val name: String,
    val minGuests: Int,
    val maxGuests: Int,
    val price: Int,
    val hall: Hall                
)

File: backend/models/offer.js
import mongoose from 'mongoose'

const offerSchema = new mongoose.Schema({
    offerImageUrl: {type:String, required:true},
    name: {type:String, required:true},
    minGuests: {type:Number, required:true},
    maxGuests: {type:Number, required:true},
    price: {type:Number, required:true},
    hall: {type: mongoose.Schema.Types.ObjectId, ref: 'Hall', required:true }
})

export default mongoose.model("Offer", offerSchema)